tasks.py

```diff
@@ -296,6 +296,12 @@ def galaxy_build(context, force=False):
         command += " --force"
     context.run(command)
 
+@task
+def force_install(context):
```

Comment:
I think we should name this `galaxy_install` to be consistent with the `galaxy_build` task. Also, any additional CLI args to the original command (`-p`, `--force`, etc) should probably be added as task kwargs (with sane defaults) as well so they show up when running `invoke galaxy-install --help`.

Reply:
Good call, will do both.

Reply:
I made `--force` optional as you suggested. I don't think we should make the output path flexible as that will break the VS Code debugger integration, which expects them to be in "collections". What do you think?

Reply:
We agreed this verbally.